src/Core/tests/Benchmarks.Droid/ImageBenchmarks.cs

```diff
@@ -70,6 +70,25 @@ public async Task ImageHelperFromFile()
 		await callback.SuccessTask;
 	}
 
+	[Benchmark]
+	public async Task ImageHelperFromFont()
+	{
+		var callback = new Callback();
+
+		handler!.Post(() =>
+		{
+			Microsoft.Maui.PlatformInterop.LoadImageFromFont(
+				context,
+				Color.Aquamarine,
+				"A",
+				Typeface.Default,
```

Comment:
If this is Android's `Typeface.Default`, you might save it in a static variable. Just to get it out of the way of the benchmark calling Java as much.